src/js/utils/css.js

```diff
@@ -58,6 +60,29 @@ define([
         }
     };
```

Comment:
Wanted to avoid making any significant changes here, but could probably add logic to to convert object to string if any field has `!important` in it.